### DISCUSSION TITLE
Error opening recording device. Error code: -9999 Unanticipated host error

### DISCUSSION THREAD

**Discussion by BondarAnatolii** [audacity/audacity#4378] [Q&A]
### Bug description

It doesn't matter which device I choose for recording.
Gets error "Error opening recording device. Error code: -9999 Unanticipated host error"

### Steps to reproduce

1) Open Audacity
2) Audio Setup -> Recording device
3) Choose any (no matter)
4) Press record button

### Expected behavior

Start recording without error

### Actual behavior

Instead of recording, an error occurs and that's it

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

### Additional context

Try version 3.2.4 and 3.2.5
Update audio driver
But nothing helped

The program worked without problems and stopped working in a flat place.

I downloaded a similar one and the recording works without problems in it.

**Comment by petersampsonaudacity** [comment]
See this page on Muse's Audacity Support site:

**https://support.audacityteam.org/troubleshooting/error-codes**
